### DISCUSSION TITLE
How can i install a NVIDIA drivers?

### DISCUSSION THREAD

**Discussion by adrielfilipetech** [elementary/triage#359] [Q&A]
Hi, how is going?

Im install ElementaryOS on my computer, i have a GTX 1080TI, and i want to edit some videos with Davinci Resolve, so...
how can i install a NVIDIA Drivers on Elementary OS?
Thanks for your help

**Comment by truefakts** [comment]
I don't have an Nvidia card so I can't confirm if this works. But I would suspect this is the process because this is how it works on almost all Ubuntu based systems:

1. Launch System Settings
2. Go to System
3. Go to Firmware
4. Select the latest Nvidia driver (non-server version)

Alternatively, this article walks you through how to install the Ubuntu graphics driver manager: https://www.fosslinux.com/1462/how-to-install-nvidia-gpu-drivers-in-elementary-os.htm

1. sudo apt install software-properties-gtk software-properties-common
2. sudo add-apt-repository ppa:graphics-drivers/ppa
3. sudo apt update
4. After it is done, head to the main menu section and search for “Software & Update,” then fire it up.
5. Select Additional Drivers
6. Select the latest Nvidia driver (non-server version)

That being said, PPAs are typically removed during OS upgrades. You might have to repeat this process if you upgrade to EOS 8

**Reply by adrielfilipetech** [reply]
Thanks!

**Comment by danirabbit** [comment]
Drivers in OS 7 are available from the Installed & Updates tab of AppCenter

In elementary OS 8 they will be available from System Settings → System → Drivers
